### PR TITLE
EPMRPP-37443: Edit attributes

### PR DESCRIPTION
**Pull request by DiscoElevator**

### CODE REVIEW

File: app/src/components/main/attributeList/editableAttribute/editableAttribute.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    onRemove: () => {},
+    onChange: () => {},
+  };
+
+  enterEditMode = () => this.props.onEdit(this.props.attribute);
+
+  exitEditMode = () => this.props.onEdit(null);
+
+  calculateFormName = (attribute) =>
+    attribute ? `attributesEditor__${attribute.key}_${attribute.value}` : 'attributesEditor';
```
Comment (AmsterGet) [inline app/src/components/main/attributeList/editableAttribute/editableAttribute.jsx]: `attribute` is `{}` by default, so you will get `attributesEditor__undefined_undefined`

File: app/src/components/main/attributeList/attributeList.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+const isEditMode = (currentAttribute, editedAttribute) =>
+  editedAttribute &&
+  currentAttribute.key === editedAttribute.key &&
+  currentAttribute.value === editedAttribute.value;
+
+export const AttributeList = ({ attributes, editedAttribute, onChange, onEdit, onAddNew }) => (
+  <Fragment>
+    {attributes
+      .filter((attr) => !attr.system)
```
Comment (AmsterGet) [inline app/src/components/main/attributeList/attributeList.jsx]: You can use `notSystemAttributePredicate` from `common/utils/attributeUtils` like in the `pages/inside/common/itemInfo/attributesBlock`

File: app/src/pages/inside/common/modals/editItemModal/editItemModal.jsx [hunk 3]
```diff
@@ -11,26 +11,18 @@ import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
...
 import { connect } from 'react-redux';
 import { fetch } from 'common/utils';
 import { URLS } from 'common/urls';
+import { AttributeListField } from 'components/main/attributeList';
 import styles from './editItemModal.scss';
 
 const cx = classNames.bind(styles);
 
 const messages = defineMessages({
   tagsLabel: {
```
Comment (AmsterGet) [inline app/src/pages/inside/common/modals/editItemModal/editItemModal.jsx]: `attributesLabel`

Comment (DiscoElevator) [reply app/src/pages/inside/common/modals/editItemModal/editItemModal.jsx]: fixed

File: app/src/components/main/attributeList/attributeList.jsx (hunk 2 above)
Comment (DiscoElevator) [reply app/src/components/main/attributeList/attributeList.jsx]: fixed

File: app/src/components/main/attributeList/editableAttribute/editableAttribute.jsx (hunk 1 above)
Comment (DiscoElevator) [reply app/src/components/main/attributeList/editableAttribute/editableAttribute.jsx]: fixed

**Pull request merged**